modules/nodes-nodes-resources-configuring-auto.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-resources-configuring.adoc
+
+[id="nodes-nodes-resources-configuring-auto_{context}"]
+= Automatically allocating resources for nodes
+
+{product-title} can automatically determine the optimal `system-reserved` CPU and memory resources for nodes associated with a specific machine config pool and update the nodes with those values when the nodes start.  
+
+To automatically determine and allocate the `system-reserved` resources on nodes, create a `KubeletConfig` custom resource (CR) to set the `autoSizingReserved: true` parameter. A script on each node calculates the optimal values for the respective reserved resources based on the installed CPU and memory capacity on each node. The script takes into account that increased capacity requires a corresponding increase in the reserved resources. 
+
+Automatically determining the optimal `system-reserved` settings ensures that your cluster is running efficiently and prevents node failure due to resource starvation of system components, such as CRI-O and kubelet, without your needing to manually calculate and update the values. 
```

Comment:
Maybe,

> without manually calculating and updating the values.

Reply:
@jboxman I had that originally. But it sounds to me that the settings don't need to manually calculate and update the values, as that is the subject of the sentence. 
What is your concern? A bit awkward?

Reply:
@mburke5678 just read different than I expected